src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
```

Comment:
you don't need create new Lottery each iteration, move this to the variable before loop and use it in loop

Reply:
It is not necessary to call toString explicitly, because in the print method toString is called implicitly on objects